tutorials/mie_potential/src/main.rs

```diff
@@ -0,0 +1,82 @@
+extern crate mie_potential;
+use mie_potential::Mie;
+
+extern crate lumol;
+use lumol::sys::{System, UnitCell, Particle};
+use lumol::energy::PairInteraction;
+use lumol::sim::{Simulation, MonteCarlo};
+use lumol::sim::mc::Translate;
+use lumol::out::{EnergyOutput, PropertiesOutput};
+use lumol::types::Vector3D;
+use lumol::units;
+
+// helper function to build a system made of 125 particles
+// interacting via a Mie potential (using LJ exponents)
+fn get_system() -> System {
+    let mut system = System::with_cell(UnitCell::cubic(28.0));
+    for i in 0..5 {
+        for j in 0..5 {
+            for k in 0..5 {
+                let mut part = Particle::new("Ar");
+                part.position = Vector3D::new(i as f64 * 3.4, j as f64 * 3.4, k as f64 * 3.4);
+                system.add_particle(part);
+            }
+        }
+    }
+    println!("Added {} atoms to the system!", system.size());
+
+    // build a potential
+    let mie = Mie::new(units::from(3.405, "A").unwrap(),
+                       units::from(1.0, "kJ/mol").unwrap(),
+                       12.0,
+                       6.0);
+    // use the potential with a cut off radius of 3 * sigma
+    let mut pairs = PairInteraction::new(Box::new(mie), 3.0 * 3.405);
+    // use tail corrections to account for our truncation
+    pairs.enable_tail_corrections();
+    // finally add this interaction to the system for Argon atoms
+    system.add_pair_potential("Ar", "Ar", pairs);
+    system
+}
+
+// This function runs a small Monte-Carlo simulation for Argon using a Mie potential.
+fn main() {
```

Comment:
I would go for a simpler main, using the existing `Input` and just adding the Mie potential:

```rust
fn main() {
    let input = Input::new("some.path.toml").unwrap().read().unwrap();
    
    let mut system = input.system;
    let mut simulation = input.simulation;

    // ...
    system.add_pair_potential("Ar", "Ar", pairs);

    simulation.run(&mut system, input.nsteps);
}
```

This allow to not show too many things at once, and make it easier to introduce the second part later (the code in `lumol-input` is doing the same as the code in main, no more).